packages/oui-navbar/src/index.js

```diff
@@ -0,0 +1,18 @@
+import KEYBOARD_KEYS from "./keyboard-keys.constant";
+import Navbar from "./navbar.component";
+import NavbarGroup from "./group/navbar-group.directive";
+import NavbarGroupService from "./group/navbar-group.service";
+import NavbarMenu from "./menu/navbar-menu.component";
+import NavbarService from "./navbar.service";
+
+angular
+    .module("oui.navbar", [
+        "ngAria",
+        "ngSanitize"
+    ])
+    .constant("KEYBOARD_KEYS", KEYBOARD_KEYS)
```

Comment:
... and put this in "oui" module so that it can be shared between all components.

Reply:
Yes, good idea. I move it to oui-angular